### DISCUSSION TITLE
disk type on the runner

### DISCUSSION THREAD

**Discussion by aouakki** [geeksblabla/blanat#208] [General]
In the runner machine, is the disk local or on network?

**Comment by aboullaite** [comment]
Local

**Reply by aouakki** [reply]
@aboullaite is it ssd? if you can share more about the maximum throughput on read, for advanced solutions this information may impact the performance if not taken in consideration. 
will be great if you can share the setup of the runner machine. 
This way anyone can replicate and verify on their side. 
Thanks in advance 🙏

**Reply by aboullaite** [reply]
I might have spoken too soon. The machine is `e2-standard-32` from Google cloud, that does not have Local SSD. https://cloud.google.com/compute/docs/general-purpose-machines#e2_machine_types Sorry for the confusion

**Reply by aouakki** [reply]
The E2 family is known for it's inconsistent performance when pushing the CPU load to it limit, which may randomly impact submissions runs, I my self tested this and first runs sometimes are slower ( no mmap use).
https://www.bigbitbus.com/2021/06/10/Google-Cloud-E2-N2-VMs/
May I suggest using the N2 to be fair.

**Reply by aboullaite** [reply]
Thanks for flagging. But unfortunately we can't at this point. That is something we are going to consider in future iterations (It's a learning initiative for us as well). But It's the same set of constraints for everyone

**Reply by sehe** [reply]
> But It's the same set of constraints for everyone

The same inconsistency for everyone makes it a lottery

**Reply by aouakki** [reply]
@aboullaite are you sure it's `e2-standard-32` ? or `e2-standard-8`?
e2-standard-32 has 32 cores and you said early that the runner has 8 cores. 
also would you mind telling what type of disk on it:
<img width="1128" alt="image" src="https://github.com/geeksblabla/blanat/assets/2286248/bc3fcf50-10fc-4b17-8102-6c32d79c4f6c">
thanks in advance 🙏

**Reply by aboullaite** [reply]
the latter
